ios/browser/api/history/brave_history_api.mm

```diff
@@ -0,0 +1,309 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ios/browser/api/history/brave_history_api.h"
+#include "brave/ios/browser/api/history/brave_browsing_history_driver_ios.h"
+#include "brave/ios/browser/api/history/brave_history_observer.h"
+#include "brave/ios/browser/api/history/history_service_listener_ios.h"
+
+#include "base/bind.h"
+#include "base/compiler_specific.h"
+#include "base/containers/adapters.h"
+#include "base/guid.h"
+#include "base/sequenced_task_runner.h"
+#include "base/strings/sys_string_conversions.h"
+#include "base/task/post_task.h"
+#include "base/task/thread_pool.h"
+
+#include "components/browsing_data/core/browsing_data_utils.h"
+#include "components/history/core/browser/browsing_history_service.h"
+#include "components/history/core/browser/history_service.h"
+#include "components/history/core/browser/history_types.h"
+#include "components/keyed_service/core/service_access_type.h"
+#include "components/sync/driver/sync_service.h"
+
+#include "ios/chrome/browser/application_context.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state_manager.h"
+#include "ios/chrome/browser/browsing_data/browsing_data_remove_mask.h"
```

Comment:
you appear to have includes that are not being used. Also see https://google.github.io/styleguide/objcguide.html#order-of-includes and all these headers should be grouped together except for `brave_history_api.h`